.cirrus.yml

```diff
@@ -270,8 +282,15 @@ qa_task:
     ffmpeg -loglevel warning -f x11grab -video_size 1280x960 -i ${DISPLAY} -codec:v libx264 -r 12 ${CIRRUS_WORKING_DIR}/recording_${IDEA_VERSION}.mp4
   run_its_script: |
     echo "Run ITs on ${IDEA_VERSION}"
+    echo "Run ITs on ${QA_CATEGORY}"
     source .cirrus/use-gradle-wrapper.sh
-    gradle :its:check --stacktrace -i -PijVersion=${IDEA_VERSION} -PslPluginDirectory=${CIRRUS_WORKING_DIR}/staged-plugin
+    if [[ ${QA_CATEGORY} = "Idea2022_Standalone" ]] || [[ ${QA_CATEGORY} = "Idea2023_Standalone" ]]; then
```

Comment:
I would check if the name ends by Standalone or Connected2, so you don't have to duplicate everything for each year